lib/configuration/command_configuration.ml

```diff
@@ -404,7 +404,9 @@ module Printer = struct
       let print_if_some output = Option.value_map output ~default:() ~f:(Format.fprintf ppf "%s@.") in
       match output_format with
       | Stdout -> Format.fprintf ppf "%s" rewritten_source
-      | Overwrite_file -> Out_channel.write_all ~data:rewritten_source (Option.value path ~default:"/dev/null")
+      | Overwrite_file ->
+        if ((List.length replacements) > 0) then
```

Comment:
`replacements <> []` is slightly shorter and much faster large lists.

Reply:
You are so right that it hurts ...